### ISSUE TITLE
Trying to execute the gui without neovim in the path produces no output

### ISSUE THREAD

**Issue by RedX2501** [equalsraf/neovim-qt#76]
When running neovim-qt.exe without neovim in the path produces no output or error messages.
Nothing at all happens, not even when trying to run from cmd.exe.

Maybe some indication as to why it is not starting would be helpful. Maybe a messagebox?

**Comment by equalsraf** [comment]
That is strange, you should see a window with an error message on top. for example something like the following (but with a different message)
![neovimqt](https://cloud.githubusercontent.com/assets/715238/11837698/7c9cec94-a3da-11e5-8319-32b026ec3309.png)

Definitely a bug from 9c86f540524f6b0c6b1131c5f1317d5e414ae559

1. The timer is not being started so the Window is not shown (this is fixed in #80)
2. The actual error should be something like "File not found" and should fail immediately but for some reason the signal is not being fired.

**Comment by equalsraf** [comment]
Correction: the error you should be seeing is this
![neovimqt](https://cloud.githubusercontent.com/assets/715238/11838507/6dd77940-a3df-11e5-901f-0bf990fa857c.png)

**Comment by equalsraf** [comment]
@RedX2501 does this version work better? https://ci.appveyor.com/project/equalsraf/neovim-qt/build/439/artifacts

**Comment by RedX2501** [comment]
Yeah much better. Now the message pop-up instantly.

How do i attach neovim-qt to an already running instance of neovim? I could not find what parameters neovim and neovim-qt need. The `--server` mentioned in the wiki does not seem to work.

**Comment by equalsraf** [comment]
@RedX2501 what is the message that you see? It should be the second one "Unable to start ..."

> How do i attach neovim-qt to an already running instance of neovim? I could not find what parameters neovim and neovim-qt need. The --server mentioned in the wiki does not seem to work.

in a nutshell for Neovim

    set NVIM_LISTEN_ADDRESS=127.0.0.1:7777
    nvim

and then

    nvim-qt --server 127.0.0.1:7777

Neovim creates a unique named pipe by default - you can override this with the environment variable NVIM_LISTEN_ADDRESS. It can be either a named pipe \\.\nvim\... or a tcp address as 127.0.0.1:7777
In Windows using a TCP socket is easier because named pipes need to named as \\.\... and there is no easy way to list all pipes without third party tools.

**Comment by RedX2501** [comment]
The message is
```
Unable to start nvim: Process failed to start: No such file or directory
```

and your concise information about attaching was spot on. Thanks

**Comment by equalsraf** [comment]
I've merged #80 into the master branch. The new packages should start building shortly.
